content/accessibility/developer-toolkit.mdx

```diff
@@ -0,0 +1,118 @@
+---
+title: Developer toolkit
```

Comment:
This page does not appear in the navigation.

Reply:
we're not including this page as this is going to live in design documentation

Reply:
@yaili do you want me to delete this page and save it else where in case it's needed in the future?

Reply:
good idea, I gave you access to the docs shared Drive, are you able to create a doc in the root and move the content there? https://drive.google.com/drive/u/2/folders/0AGa0lMGP0Hj5Uk9PVA 😄

Reply:
Done 🙂 https://docs.google.com/document/d/1OtV0TvrqkSmtovK4Boo2R9jYnC0a0HC4mcBJYyZ4r0o/edit?usp=sharing